### PR TITLE
Add metadata and schema

### LINKED ISSUE #228: SEO: Add meta tags
As @benalleng pointed out in [this comment](https://github.com/saving-satoshi/saving-satoshi/pull/199#issuecomment-1454088002) we are missing meta tags that will help improve our listing in search engines.

We should make sure that on every page we include the proper [meta tags](https://ahrefs.com/blog/seo-meta-tags/) + [og meta tags](https://ahrefs.com/blog/open-graph-meta-tags/). Perhaps @ecurrencyhodler / @satsie / @GBKS can start thinking about what would be the proper content for the tags.

### PR DESCRIPTION
**Pull request by ecurrencyhodler**
Closes #228

## Pull Request checklist

Please review the below PR requirements:

- Build was run locally and any changes were pushed
- Ensure that the title clearly describes the changes
- Issue to be referenced in the PR description rather than in the commits or PR title
- Clean commit history

### CODE REVIEW

File: app/head.tsx [hunk 1]
```diff
@@ -61,33 +61,42 @@ export default async function Head() {
...
-      <meta
-        property="twitter:title"
-        content="Explore the mysteries of Satoshi and learn about bitcoin along the way."
-      />
-      <meta
-        property="twitter:image"
-        content="https://savingsatoshi.com/_next/image?url=%2Fassets%2Fpreviews%2Fmain-image-preview.jpg&w=3840&q=100"
-      />
+//Schema      
+<script type="application/ld+json" class="schemantra">{
```
Comment (benalleng) [inline app/head.tsx:75]:
``` suggestion
<script type="application/ld+json" className="schemantra">
```

File: app/head.tsx [hunk 2]
```diff
@@ -61,33 +61,42 @@ export default async function Head() {
...
+  "genre": "Education Platform",
+  "headline": "Code your way through the mysteries of bitcoin!",
+  "isAccessibleForFree": true,
+  "isFamilyFriendly": true,
+  "keywords": "bitcoin, bitcoin developer, bitcoin development, btc",
+  "learningResourceType": "Coding game platform",
+  "name": "Saving Satoshi",
+  "teaches": "Bitcoin, coding",
+  "thumbnailUrl": "https://imagedelivery.net/wyrwp3c-j0gDDUWgnE7lig/6818cd1b-ea0d-49a4-ea14-d34c1bbae100/public"
+}</script>
```
Comment (benalleng) [inline app/head.tsx:99]:
``` suggestion
</script>
```

**Review by benalleng — CHANGES_REQUESTED** [review]
Few small changes!

**Comment by ecurrencyhodler** [comment]
Thanks ben! I think that worked. 💪

**Review by benalleng — APPROVED** [review]

**Comment by ecurrencyhodler** [comment]
Hmmm the pr is creating an error when the page loads. I think I should close the pr and let someone else handle this.

https://user-images.githubusercontent.com/32662508/233157770-c5d0cd9a-9dd3-4665-8267-723b1e13ce43.mov

**Comment by benalleng** [comment]
It looks like Next13 has a new metadata format, as sepcified in the docs [here](https://beta.nextjs.org/docs/upgrade-guide#step-3-migrating-nexthead) and [here](https://beta.nextjs.org/docs/api-reference/metadata). I'll take a look at how they work tomorrow

**Pull request closed without merging**